platform-tests/example-apps/healthcheck/redis.go

```diff
@@ -30,6 +31,10 @@ func testRedisConnection() error {
 		return errors.Wrap(err, "failed to parse VCAP_SERVICES")
 	}
 
+	if !strings.HasPrefix(credentials.URI, "rediss://") {
```

Comment:
Is that correct? 🤔

Reply:
Yup `http` => `https`. `redis` => `rediss`.

Reply:
https://www.iana.org/assignments/uri-schemes/prov/rediss